connector-model/src/main/resources/connector-definition-descriptor.xsd

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xsd:schema xmlns="http://www.bonitasoft.org/ns/connector/definition/6.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.bonitasoft.org/ns/connector/definition/6.0">
```

Comment:
this is not the proper version. Should be `http://www.bonitasoft.org/ns/connector/definition/6.1`

Reply:
We also need to update it here https://github.com/bonitasoft/bonita-process-model/blob/develop/bundles/org.bonitasoft.bonita2bar/src/org/bonitasoft/bonita2bar/process/ProcessRegistryImpl.java#L67

Reply:
This is like this on purpose for now. The migration in proc file is not done properly and we still have references to the old version.
https://bonitasoft.atlassian.net/browse/DEV-455 should tackle this.